earn/src/data/MarketInfo.ts

```diff
@@ -0,0 +1,57 @@
+import Big from 'big.js';
+import { secondsInYear } from 'date-fns';
+import { ethers } from 'ethers';
+import { Address } from 'wagmi';
+
+import { toBig, toImpreciseNumber } from '../util/Numbers';
+
+export type MarketInfo = {
+  lender0: Address;
+  lender1: Address;
+  borrowerAPR0: number;
+  borrowerAPR1: number;
+  lender0Utilization: number;
+  lender1Utilization: number;
+  lender0TotalAssets: Big;
+  lender1TotalAssets: Big;
+  lender0TotalBorrows: Big;
+  lender1TotalBorrows: Big;
+  lender0AvailableAssets: Big;
+  lender1AvailableAssets: Big;
+};
```

Comment:
Are the changes to this data structure meant to match Prime? Or is this entirely new?

Reply:
I just thought it would be better to give all the info since you removed some on prime that we use on here, so instead of having to recalculate the values using `lender0AvailableAssets` and `lender0TotalAssets`, I thought it would be easier also to include `lender0TotalBorrows`.